### PR TITLE
ppss.kr - ad

### PR DESCRIPTION
**Pull request by JobcenterTycoon**
`https://ppss.kr/` ad picture

Unbenannt

### CODE REVIEW

File: filters-share/specific_URL.txt [hunk 1]
```diff
@@ -382,6 +382,7 @@
 /^https:\/\/thumb\.toomics\.com\/upload\/banner\/(?!main|cut)/
 ||ppss.kr/wp-content/uploads/2021/09/ktx_
 ||ppss.kr/wp-content/uploads/2022/02/cartax-
+||ppss.kr/wp-content/uploads/2022/10/pacey
```
Comment (piquark6046) [inline filters-share/specific_URL.txt]: Please move to top of the files.

Comment (piquark6046) [reply filters-share/specific_URL.txt]: And, update its version.

**Review by piquark6046 — CHANGES_REQUESTED** [review]

File: filters-share/specific_URL.txt (hunk 1 above)
Comment (JobcenterTycoon) [reply filters-share/specific_URL.txt]: All 3 filter or just the new one ?

Comment (piquark6046) [reply filters-share/specific_URL.txt]: New one.

Comment (JobcenterTycoon) [reply filters-share/specific_URL.txt]: i opened a new one because too many conflicts are here

**Pull request closed without merging**